File: support_tools/swigwin-3.0.12/Examples/test-suite/octave/input_runme.m
# do not dump Octave core
if exist("crash_dumps_octave_core", "builtin")
  crash_dumps_octave_core(0);
endif

input

f = Foo();
if (f.foo(2) != 4)
  error
endif

try
  a=f.foo();
  error
catch
end_try_catch

if (!strcmp(sfoo("Hello"),"Hello world"))
  error
endif

try
  a=sfoo();
  error
catch
end_try_catch

File: support_tools/swigwin-3.0.12/Examples/test-suite/python/input_runme.py
from input import *

f = Foo()
if f.foo(2) != 4:
    raise RuntimeError

if f.foo(None) != None:
    raise RuntimeError

if f.foo() != None:
    raise RuntimeError

if sfoo("Hello") != "Hello world":
    raise RuntimeError

if sfoo(None) != None:
    raise RuntimeError

if sfoo() != None:
    raise RuntimeError
